### DISCUSSION TITLE
Raw file not accessible after push

### DISCUSSION THREAD

**Discussion by nikiibarbaro** [community/community#22475] [Projects and Issues]
<p>Hey all,</p>
<p>today I encountered a problem where I can’t identify, if it’s my fault because I missed something or if it’s an GitHub error.</p>
<p>To specify my problem: I need to store a md5 file which contains an md5 hash, after pushing it to my public repository, it’s there, but as soon as I want to view it in raw format, I get the “404: Not Found” status code and I don’t know why, I also tried it with different md5 files and as soon as I’m changing the file to “foo.md5” it works.</p>
<p>Has anyone a solution to my problem?</p>
<p>For reference the link to the file which doesn’t work in raw mode: <a href="https://raw.githubusercontent.com/nikiibarbaro/repository.xpress/master/zips/addons.xml.md5" rel="noopener nofollow ugc">https://raw.githubusercontent.com/nikiibarbaro/repository.xpress/master/zips/addons.xml.md5</a></p>
<p>And here the tree view: <a href="https://github.com/nikiibarbaro/repository.xpress/blob/master/zips/addons.xml.md5" class="inline-onebox" rel="noopener nofollow ugc">repository.xpress/addons.xml.md5 at master · nikiibarbaro/repository.xpress · GitHub</a></p>

**Comment by jainamoswal** [comment]
<p>I can see raw content.<br>
Try to clear your browser cache and retry.<br>
It should work then.</p>

**Comment by nikiibarbaro** [comment]
<p>Sorry I messed something up with the link, I edited the link, does this link work for you aswell?</p>

**Comment by nikiibarbaro** [comment, marked as answer]
<p>So as I understood the problem was that I had pushed too many times in short time periods so GitHub couldn’t handle my requests</p>
